diff --git a/components/omnibox/browser/history_url_provider.cc b/components/omnibox/browser/history_url_provider.cc
index 526c06fdfe..16135f689c 100644
--- a/components/omnibox/browser/history_url_provider.cc
+++ b/components/omnibox/browser/history_url_provider.cc
@@ -490,6 +490,8 @@ void HistoryURLProvider::Start(const AutocompleteInput& input,
   if (base::CommandLine::ForCurrentProcess()->HasSwitch("omnibox-autocomplete-filtering"))
     return;
 
+  if (base::CommandLine::ForCurrentProcess()->HasSwitch("scope-omnibox-autocomplete"))
+    return;
   // We'll need the history service to run both passes, so try to obtain it.
   history::HistoryService* const history_service =
       client()->GetHistoryService();
diff --git a/components/url_formatter/url_fixer.cc b/components/url_formatter/url_fixer.cc
index 30cc94396a..6a18225b86 100644
--- a/components/url_formatter/url_fixer.cc
+++ b/components/url_formatter/url_fixer.cc
@@ -647,7 +647,17 @@ GURL FixupURLInternal(const std::string& text,
     }
     if (chrome_url && !parts.host.is_valid()) {
      if (!base::CommandLine::ForCurrentProcess()->HasSwitch("omnibox-autocomplete-filtering") ||
-         base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("omnibox-autocomplete-filtering").contains("chrome"))
+         base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("omnibox-autocomplete-filtering").contains("chrome")) {
+      const std::string kOmniboxScopeFlag = "scope-omnibox-autocomplete";
+      const base::CommandLine* cmdline = base::CommandLine::ForCurrentProcess();
+      if (cmdline->HasSwitch(kOmniboxScopeFlag)) {
+        std::vector<std::string> switch_values = base::SplitString(
+            cmdline->GetSwitchValueASCII(kOmniboxScopeFlag), ",",
+            base::KEEP_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
+        if (!std::ranges::contains(switch_values, "chrome"))
+          return GURL();
+      }
+     }
       url.append(kChromeUIDefaultHost);
     }
     FixupPort(trimmed, parts.port, &url);
